app/models/user.rb

```diff
@@ -760,7 +760,7 @@ def total_unread_notifications
   end
 
   def reviewable_count
-    Reviewable.list_for(self, include_claimed_by_others: !redesigned_user_menu_enabled?).count
+    Reviewable.list_for(self).count
```

Comment:
The default for `include_claimed_by_others` is true, so I think we need to hard-code false here to keep the correct behaviour ?
```suggestion
    Reviewable.list_for(self, include_claimed_by_others: false).count
```